l10n_mx_sat_account/README.rst

```diff
@@ -0,0 +1,99 @@
+=========================
+Sale Maxico SAT Reference
+=========================
```

Comment:
This is what is done on the **community** l10n_mx. Why are you re-doing it?

Reply:
You just need to build your own report.

https://github.com/odoo/odoo/tree/13.0/addons/l10n_mx/data

The data is there as tags and as accounts.

Reply:
@nhomar Because tags are many2many so you would be reporting the same account twice.

But I am happy to be wrong...